vivisect/symboliks/common.py

```diff
@@ -81,6 +81,7 @@ def __init__(self):
         self.kids          = []
         self.parents       = []
         self.cache         = {}
+        self._reduced       = False
```

Comment:
i like the idea.  need to factor the cost versus benefit, and keep in mind that reduce doesn't use walkTree.  they're two different animals.